### PR TITLE
chore: fix linter issues

### PR DESCRIPTION
**Pull request by xcoulon**
also, disable rule on dot-imorts (ie, make always permitted)

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: pkg/status/toolchaincluster_test.go [hunk 1]
```diff
@@ -42,7 +43,7 @@ func TestGetToolchainClusterConditions(t *testing.T) {
 			err := ValidateComponentConditionReady(conditions...)
 
 			// then
-			assert.NoError(t, err)
+			require.NoError(t, err)
```
Comment (MatousJobanek) [inline pkg/status/toolchaincluster_test.go:46]: may I know why there are all these changes for replacing `assert` with `require`?

Comment (xcoulon) [reply pkg/status/toolchaincluster_test.go:46]: that was a feedback from ["testifylint"](https://golangci-lint.run/usage/linters/#testifylint):
```
require-error: for error assertions use require (testifylint)
```
If it's an issue for us, then we could probably try to disable this check 🤷‍♂️

Comment (MatousJobanek) [reply pkg/status/toolchaincluster_test.go:46]: ok, that makes sense. It's fine - let's keep it there :+1:

**Review by MatousJobanek — COMMENTED** [review]
Thanks :1st_place_medal:

**Review by MatousJobanek — APPROVED** [review]
And approve :smile:

**Comment by xcoulon** [comment]
> And approve 😄

thanks!

**Pull request merged**